apps/hooks-docs/src/components/Input.tsx

```diff
@@ -81,6 +81,7 @@ export const Input = <
           min={props.min}
           max={props.max}
           pattern={props.pattern}
+          // @ts-ignore
```

Comment:
question: why the TS ignore

Reply:
the `inputmode` attribute is not yet added to kaioken's DOM types.

Reply:
ah okay! thanks